docs/dns.md

```diff
@@ -1,12 +1,11 @@
 # DNS Support
-If you point a domain or subdomain to Google Cloud DNS, you can use `kctf-setup-dns` and `kctf-chal-dns` to map *[challenge_name].yourdomain.com* to your challenge.
+If you point a domain or subdomain to Google Cloud DNS as an NS record, kCTF creates a DNS zone that maps the challenge to your challenge's IP address.
 
 Follow these steps:
 
-1. Configure the subdomain in `kctf-setup-config-create`. 
+1. Configure the domain in `kctf-setup-config-create --domain-name foo.example.com`. 
 
-   Example: If you enter *foo.example.com*, the challenges will be deployed on *[challenge-name].foo.example.com*.
-1. Run `kctf-setup-dns` to create the DNS zone in Google Cloud DNS.
+   Example: If you enter *foo.example.com*, the challenges are be deployed on *xxxxx.foo.example.com*.
+1. Run `kctf-cluster-start`
+1. Add `DNS=true` on `config/challenge.conf`.
 1. Point the subdomain to Google Cloud DNS. This requires [adding an NS record on your name server](https://cloud.google.com/dns/docs/update-name-servers).
```

Comment:
That page tells you to run `gcloud dns managed-zones describe $ZONE`, but it's clear what ZONE to use.
I.e. I found it with `gcloud dns managed-zones list` IIRC.

Reply:
k